schemas/answ_action_response.fbs

```diff
@@ -0,0 +1,17 @@
+// A message for holding information on the (immediate) outcome of a command to (or state change of) a filewriter
+
+file_identifier "answ";
+
+enum ActionType : byte { StartJob = 0, SetStopTime, StopNow, HasStopped }
+enum ActionOutcome : byte { Success = 0, Failure }
+
+table ActionResponse {
+    service_id : string (required);       // The identifier for the instance of the file-writer that responded to the command
+    job_id : string (required);           // A unique identifier for the file writing job
+    action : ActionType;                  // What type of command is this a response to?
+    outcome : ActionOutcome;              // Was the command accepted (action successfull)?
```

Comment:
Would a boolean not be easier?

Reply:
It would be slightly easier in principle. However, seeing as I have already started writing code using this implementation it would actually be a bit harder. Can still change it if you prefer.